### DISCUSSION TITLE
error with apnormalize

### DISCUSSION THREAD

**Discussion by TaurusOrion11** [iraf-community/iraf#248] [Q&A]
Hello,

I am trying to use the task apnormalize with iraf 2.17, but it fails with an error like below.
```
ERROR: ambiguous task 'apnorm'
    called as: `apnormalize (input=...
```
It both happened with the macOS64 and linux64 environments. I did not have any problems with iraf 2.16.
Could you please help me?

**Comment by olebole** [comment]
Can you provide a minimal script that shows this problem?

**Comment by TaurusOrion11** [comment]
Thank you for the reply. I have executed the task like

apnormalize flat nflat refer=[aptracedfile] find- recenter- resize+ edit+ trace- fittrac- normali+ fitspec+ order=5 niterate=5 low_rej=3 high_rej=3

**Reply by olebole** [reply]
I mean: Please provide a minimal example that, when run, produces this error. Here it is important which packages you load and in which order.

**Reply by TaurusOrion11** [reply]
I am using a cl script that worked with previous iraf versions, which calls the packages below;
images, tv, noao, RV, plot, proto, twodspec, apextract, longslit, onedspec, utilities, imutil, imred, ccdred, noao, artdata, echelle

After I got your comment, I tried using apnormalize with only running echelle package, and that worked. 
I also tried to use the apnormalize after running the package listed above manually, but the apnormalize worked as usual.
Although I still don't know why the problem happens when using the cl script, thank you very much for your help.
